actions/admin/site/update_advanced.php

```diff
@@ -45,6 +45,19 @@
 	elgg_disable_simplecache();
 }
 
+$cache_symlinked = is_dir(elgg_get_root_path() . 'cache/');
+if ('on' === get_input('cache_symlink_enabled') && !$cache_symlinked) {
+	$cache_symlinked = symlink(elgg_get_data_path() . 'views_simplecache/', elgg_get_root_path() . 'cache/');
```

Comment:
Use `elgg_get_cache_path()`. Better yet, we should make SimpleCache::getPath if it doesn't already exist.